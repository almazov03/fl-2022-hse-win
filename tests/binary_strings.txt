STARTING_NON_TERMINAL = $S$;
$S$ = { "1" $S$ }; $S$ = { $T$ };
$T$ = { "0" $T$ }; $T$ = { EPS };