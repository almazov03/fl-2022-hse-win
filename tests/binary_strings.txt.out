LexToken(START,'STARTING_NON_TERMINAL',1,0)
LexToken(EQUAL,'=',1,22)
LexToken(NON_TERMINAL,'S',1,24)
LexToken(SEPARATOR,';',1,27)
LexToken(NON_TERMINAL,'S',2,30)
LexToken(EQUAL,'=',2,34)
LexToken(LBR,'{',2,36)
LexToken(TERMINAL,'1',2,38)
LexToken(NON_TERMINAL,'S',2,42)
LexToken(RBR,'}',2,46)
LexToken(SEPARATOR,';',2,47)
LexToken(NON_TERMINAL,'S',2,49)
LexToken(EQUAL,'=',2,53)
LexToken(LBR,'{',2,55)
LexToken(NON_TERMINAL,'T',2,57)
LexToken(RBR,'}',2,61)
LexToken(SEPARATOR,';',2,62)
LexToken(NON_TERMINAL,'T',3,65)
LexToken(EQUAL,'=',3,69)
LexToken(LBR,'{',3,71)
LexToken(TERMINAL,'0',3,73)
LexToken(NON_TERMINAL,'T',3,77)
LexToken(RBR,'}',3,81)
LexToken(SEPARATOR,';',3,82)
LexToken(NON_TERMINAL,'T',3,84)
LexToken(EQUAL,'=',3,88)
LexToken(LBR,'{',3,90)
LexToken(EMPTY,'EPS',3,92)
LexToken(RBR,'}',3,96)
LexToken(SEPARATOR,';',3,97)
